tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -189,6 +189,72 @@ def test_slurm_ticket_17399(
     )
 
 
+@pytest.mark.usefixtures("instance", "os")
+def test_slurm_from_login_nodes_in_private_network(
+    region,
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    architecture,
+    scheduler_commands_factory,
+    vpc_stack,
+):
+    """Test Slurm features in login nodes inside a private network."""
+
+    scaledown_idletime = 3
```

Comment:
I thought about this and didn't do it for two reasons:
* I wanted to avoid a combinatorial growth of conditions and test cases 
* I wanted to avoid making our tests more and more complex and hard to decode

Reply:
This might limit how the Slurm tests evolve since it won't be easy to remember updating them in multiple code blocks. I however agree that we should strike a balance on the combination of parameters used in each test. Since this test simply attempts to cover core functionality of using Slurm it should be fine (even if the other test might evolve).

Reply:
We should (and are planning to) achieve the same result (test code reusability) but with a different approach that doesn't require make each existing test more and more convoluted and hard to understand.